napari_allencell_annotator/view/images_view.py

```diff
@@ -285,6 +285,7 @@ def _handle_shuffle_clicked(self) -> None:
             Toggle state of the shuffle button.
         """
         new_toggle_state: bool = not self._annotator_model.is_images_shuffled()
+        self.viewer.clear_layers()
```

Comment:
could this also trigger within the viewer when the `images_shuffled` signal is emitted?

Reply:
connected to images_shuffled in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/32513324d69c73e339295e791e5f9ee573025fef